types/jest-plugin-context/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for jest-plugin-context 2.9
+// Project: https://github.com/negativetwelve/jest-plugins/tree/master/packages/jest-plugin-context
+// Definitions by: Aneil Mallavarapu <https://github.com/aneilbaboo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/// <reference types="jest" />
+
+declare module 'jest-plugin-context' {
```

Comment:
This isnt the correct way to declare this as module and global. See section https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module and update accordingly.

Reply:
If I make those changes. I.e.:

```typescript
// Type definitions for jest-plugin-context 2.9
// Project: https://github.com/negativetwelve/jest-plugins/tree/master/packages/jest-plugin-context
// Definitions by: Aneil Mallavarapu <https://github.com/aneilbaboo>
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
// TypeScript Version: 2.3

/// <reference types="jest" />

declare global {
  export const context: jest.Describe;
}

export default context;
```


I get the following errors:
```shell
$ npm run lint jest-plugin-context
Error: /Users/aneil/code/DefinitelyTyped/types/jest-plugin-context/index.d.ts:9:1
ERROR: 9:1   strict-export-declare-modifiers  Prefer 'export' to 'declare' in an external module. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md
ERROR: 13:1  strict-export-declare-modifiers  Prefer to explicitly write 'export' for an external module export. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md
```

Reply:
Any thoughts on how / whether to address this?